src/main/java/com/gameList/gamelist/GameListController.java

```diff
@@ -53,7 +51,7 @@ public GameListController(GameListService theGameList) {
 	 * SETUP NEEDED:
 	 * Note: it has the same admin and password we used in class
 	 * 1. On phpMyAdmin you will have to make a database called "game_list" and a table called "gamelist"
-	 * 2. this table will have 4 columns id(primary key + auto increment), gamename, console and year
+	 * 2. this table will have 5 columns id(primary key + auto increment), gamename, console, description, and year
```

Comment:
You guys forgot to add the description column. The application doesn't compile without it!